m4/tlog/play_conf_schema.m4

```diff
@@ -92,6 +92,12 @@ M4_PARAM(`/es', `query', `file-',
          `STRING is the ', `The ',
          `M4_LINES(`query string to send to Elasticsearch')')m4_dnl
 m4_dnl
+M4_PARAM(`/es', `verbose', `opts-',
+         `M4_TYPE_BOOL(false)', true,
+         `', `', `Enable verbose output on ElasticSearch HTTP client',
```

Comment:
Even though I personally like "ElasticSearch" better, it's spelled "Elasticsearch" (with lowercase 's').